package.json

```diff
@@ -25,6 +25,7 @@
 		"bundle-analysis:run": "flub run bundleStats --dangerfile build-tools/packages/build-cli/lib/lib/dangerfile.js",
 		"ci:build": "npm run build:genver && lerna run build:compile --stream",
 		"ci:build:docs": "lerna run ci:build:docs --stream --parallel",
+		"ci:build:internal-api-docs": "lerna run ci:build:internal-api-docs --stream --parallel",
```

Comment:
Personally, I prefer using pnpm as a task runner over lerna, partially because we can migrate away from needing lerna.

This i'd prefer using `pnpm -r` like I did for "typetests:gen" below.